spec/interactors/work/refresh_metadata_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe Work::RefreshMetadata do
+  let(:owner) { User.find_by(login: OWNER) }
+  let(:collection) { create(:collection, owner_user_id: owner.id) }
+  let(:original_metadata) { [{ label: 'refresh-metadata-test', value: 'RFSH-M3DA' }].to_json }
+  let(:at_id) { 'http://example.com/manifest' }
+  let(:v3_hash) do
+    {
+      id: at_id,
+      label: {
+        values: ['refresh-metadata-test']
+      },
+      metadata: original_metadata
```

Comment:
I'm not quite sure that the `label` is correct.: the key should be a language code rather than `values`.  See https://iiif.io/api/presentation/3.0/#label

Reply:
I can change this. Thanks for clarifying